sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalProcessStreamConsumer.java

```diff
@@ -21,47 +21,69 @@
 
 import java.io.BufferedReader;
 import java.io.IOException;
+import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
-import java.util.concurrent.atomic.AtomicBoolean;
+import org.sonar.api.Startable;
 import org.sonar.api.batch.BatchSide;
+import org.sonar.api.batch.InstantiationStrategy;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.api.batch.InstantiationStrategy'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-443&issues=AWEjKJXegvt_XO9IEgZU&open=AWEjKJXegvt_XO9IEgZU)

[ISSUE_KEY]: AWEjKJXegvt_XO9IEgZU